charts/handbrake/1.0.0/values.yaml

```diff
@@ -0,0 +1,128 @@
+image:
+  repository: jlesage/handbrake
+  tag: v1.23.1
+  pullPolicy: IfNotPresent
+
+strategy:
+  type: Recreate
+
+service:
+  port:
+    port: 5800            #Required for web GUI
+
+#All values here are set as the docker defaults.
+envTpl:
+# Permissions Settings
+  USER_ID: "{{ .Values.PUID }}"
+  GROUP_ID: "{{ .Values.PGID }}"
+env:
+# General Settings
+  KEEP_APP_RUNNING: "0"
+  CLEAN_TMP_DIR: "1"
+# GUI Settings
+  DISPLAY_WIDTH: "1280"
+  DISPLAY_HEIGHT: "768"
+  SECURE_CONNECTION: "0"
+  VNC_PASSWORD:
+# Automated Conversion Preset
+  AUTOMATED_CONVERSION_PRESET: 'General/Very Fast 1080p30'
+  AUTOMATED_CONVERSION_FORMAT: 'mp4'
+  AUTOMATED_CONVERSION_KEEP_SOURCE: "1"
+  AUTOMATED_CONVERSION_NON_VIDEO_FILE_ACTION: 'ignore'
```

Comment:
Be aware: None of these get used on install.
the complete "env:" section gets overridden by questions.yaml

Reply:
They do get used in test_values.yaml, because that one ignores questions.yaml]

Reply:
Yes, I know, this is the dockers suggested default values,
Usually this values are changed by the user. 
I set the on values, in case the ui change or user deletes values from text box.
So the app have some sane defaults

Reply:
These are not defaults, let me explain:
If you added the "env" section in questions.yaml, these ALL get deleted. Regardless of what is in the 'env' section in questions.yaml

Reply:
I've manually verified and those are NOT all included in questions.yaml.
Thats not going to work (well).